sklearn/metrics/pairwise.py

```diff
@@ -582,6 +584,10 @@ def _argmin_min_reduce(dist, start):
     return indices, values
 
 
+def _argmin_reduce(dist, start):
```

Comment:
`start` is not used in the function body.

Reply:
Yes, but this is needed to match the API.

Reply:
Shouldn't it throw a linter error?

Reply:
> Shouldn't it throw a linter error?

I don't see why it should.

Sometimes a concrete implementation of a public API spec needs to ignore some arguments on purpose and it's fine. Maybe we could just have an inline comment to explain that the `start` argument is part of the `pairwise_distances_chunked` API but is it not needed for this specific reducer implementation.

Reply:
:+1:

Reply:
bd02da0 documents it.